### PR TITLE
Refactoring party

### PR DESCRIPTION
**Pull request by cheljoh**
@rrgayhart @brianrip

What's this PR do?
- Broke functions into different files. Refactored the portion of the app that renders personality results card (was crazy long html before).

Where should the reviewer start?
- purrrsonality_test.js has the original code. The file was eventually deleted. 

How should this be manually tested?
- go to localhost:3000 and take the test. Make sure the results are rendered correctly and there are no obvious bugs. 

Any background context you want to provide?
- Initially difficult to refactor because of the styling. Eventually had to make new divs and append the cards there. 

Screenshots (if appropriate)

What gif best describes this PR or how it makes you feel? 

<iframe src="//giphy.com/embed/Ti3i7TumKRkdi" width="480" height="264" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="http://giphy.com/gifs/crazy-laughing-Ti3i7TumKRkdi">Gizmo</a></p>

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+    <h1>Purrrsonality Test</h1>
+    <button name="get-questions" id="get-questions"class="button btn deep-purple lighten-3">Take the Test!</button>
+
+    <br /><%= render partial: "shared/loader_gif" %>
     <div class="left-align information">
       <div class="questions">
       </div>
     </div>
+    <div id="results1" class=row>
+
```
Comment (brianrip) [inline app/views/home/index.html.erb]: remove whitespace

File: app/assets/javascripts/helpers.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+var minQuestion = 0;
+var maxQuestion = 9;
+var questions = [];
+
+
+function hideGetQuestionsButton(){
+  $("button[name=get-questions]").hide();
+}
+
+// function showResults(response) {
```
Comment (brianrip) [inline app/assets/javascripts/helpers.js]: Is there a reason to save the commented out code?

File: app/assets/javascripts/content.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+}
+
+function grumpyCat(){
+  return  "<div class='col s6'>" +
+      "<div class='card-panel'>" +
+        "<image src='/assets/grumpy_cat.png'> <b>You - 0/50<b></image>" +
+      "</div>" +
+    "</div>";
+}
+function card(attribute, title, color, description){
```
Comment (brianrip) [inline app/assets/javascripts/content.js:167]: Great job drying out the card view!

File: app/assets/javascripts/content.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+}
+
+function facebookButton(){
+  $(".facebook")
+  .append(
+    "<br><div class='center-align'><button class='facebook-button button btn cyan accent-4'>Share on Facebook!</button></div><br>");
+}
+
+function getCat(response){
+  var cat = {};
```
Comment (brianrip) [inline app/assets/javascripts/content.js:98]: Perhaps create functions that return the different cats
function keyboardCat(){
return  { name: "Keyboard Cat", picture: "<iframe src='//giphy.com/embed/10RhccNxPSaglW' width='480' height='348' frameBorder='0' class='giphy-embed' allowFullScreen></iframe><p><a href='http://giphy.com/gifs/keyboard-cat-10RhccNxPSaglW'>via GIPHY</a></p>", description: "You are the life of the party! You also enjoy playing the keyboard and wearing blue shirts." };
}

File: app/views/home/index.html.erb (hunk 1 above)
Comment (cheljoh) [inline app/views/home/index.html.erb]: Good eye!

File: app/assets/javascripts/content.js (hunk 4 above)
Comment (cheljoh) [inline app/assets/javascripts/content.js:98]: Made changes, thanks! Good eye.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (cheljoh) [inline app/views/home/index.html.erb]: Got it!

**Comment by brianrip** [comment]
Great changes. I would say this is good to go! 👍🏻

**Comment by rrgayhart** [comment]
Great work Chelsea & Brian! 🚢 :shipit: : 🚀

**Pull request merged**